en/includes/guides/organization-management/inheritance-in-organizations/attribute-inheritance.md

```diff
@@ -1,59 +1,90 @@
+This section explains the inheritance mechanism for attributes, mappings, and dialects across organizations.
 
-You can find these configurations in the **Attribute Mappings** tab of an attribute.
+### User attributes
 
-### Mapped attributes
+- Child organizations inherit both the system-defined and custom attributes from the root organization.
 
-{% if product_name == "Asgardeo" %}
+- Only the root organization can created custom attributes.
 
-You can edit mappings related to user stores ( **MY USER STORE** in this example) independently for each organization.
+Organization administrators can access inherited user attributes from the {{product_name}} Console under **User Attributes & Stores** > **User Attributes**.
 
-![Attribute mappings]({{base_path}}/assets/img/guides/organization/attributes/b2b-edit-attribute-mappings.png){: width="700" style="display: block; margin: 0;"}
+### User attribute mappings
 
-{% else %}
+Each user store in an organization maintains mappings for user attributes. Inheritance of user attribute mappings work in the following way.
 
-You can edit the **PRIMARY** user store mappings shown below only in the root organization, which manages the primary user store shared across organizations.
+{% if product_name == "WSO2 Identity Server" %}
+- **Primary user store**
 
-However, you can edit mappings related to secondary user stores (**MY USER STORE** in this example) independently for each organization.
+    - Only the root organization can edit the user mappings for the primary user store.
 
-![Attribute mappings]({{base_path}}/assets/img/guides/organization/attributes/b2b-edit-attribute-mappings.png){: width="700" style="display: block; margin: 0;"}
+    - Child organizations inherit primary user store mappings from the root organization.
 
-{% endif %}
+- **Secondary user stores**
+
+    - Child organizations can onboard their own secondary user stores.
+
+    - Child organizations have full control over attribute mappings for secondary user stores, including:
+
+        - Editing mappings for user attributes inherited by the root organization.
+
+        - Whether to enable multi-valued user attributes (e.g. emailAddresses) for the secondary user stores.
 
-### Enabling attributes for a user store
+    The following diagram illustrates how inheritance work for the multi-valued `emailAddresses` attribute.
 
-For certain multi-valued attributes, such as **Email Addresses**, organizations can configure whether the attribute should be enabled for a specific user store as seen in the above image. For secondary user stores, you can independently manage this setting within each organization.
+    ![Attribute mappings]({{base_path}}/assets/img/guides/organization/attributes/b2b-edit-attribute-mappings.png){: width="700" style="display: block; margin: 0;"}
```

Comment:
The attached image is a screenshot of the configuration page UI. This is the same UI that has been there since before inheritance was introduced and does not illustrate the inheritance behaviour.